WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderPostUiStateBuilder.kt

```diff
@@ -175,18 +174,21 @@ class ReaderPostUiStateBuilder @Inject constructor(
     }
 
     suspend fun mapRecommendedBlogsToReaderRecommendedBlogsCardUiState(
-        recommendedBlogs: List<ReaderCardRecommendedBlog>,
-        onItemClicked: (Long, Long?) -> Unit
+        recommendedBlogs: List<ReaderBlog>,
+        onItemClicked: (Long, Long) -> Unit,
+        onFollowClicked: (ReaderRecommendedBlogUiState) -> Unit
     ): ReaderRecommendedBlogsCardUiState = withContext(bgDispatcher) {
         recommendedBlogs.take(READER_RECOMMENDED_BLOGS_LIST_SIZE_LIMIT)
                 .map {
                     ReaderRecommendedBlogUiState(
-                            name = StringEscapeUtils.unescapeHtml4(it.name),
+                            name = it.name,
```

Comment:
Just so I get a full context - how come we can remove this "unescapeHtml"?

Reply:
Previously I used Gson without any kind of custom adapter, so I've decided to unescape the String purely in the presentation layer. Now that I reused the existing model `ReaderBlog` and I'm parsing the JSON manually I've resued the existing utility class `JSONUtils.getStringDecoded` in the data layer (this is how it's done in other places)